internal/config/features.go

```diff
@@ -31,53 +33,57 @@ type features struct {
 	MOFED    *feature `toml:"mofed,omitempty"`
 	NVSWITCH *feature `toml:"nvswitch,omitempty"`
 	GDRCopy  *feature `toml:"gdrcopy,omitempty"`
+	// DisableImexChannelCreation ensures that the implicit creation of
+	// requested IMEX channels is skipped when invoking the nvidia-container-cli.
+	DisableImexChannelCreation *feature `toml:"disable-imex-channel-creation,omitempty"`
```

Comment:
For consistency, we should name this as `IMEXChannel` imo

Reply:
Do you mean `DisableIMEXChannelCreation`? (as in change the case to `IMEX`?

If you mean `IMEXChannel`, I don't agree. The intent of a feature called `IMEXChannel` as this feature sets a mode in libnvidia-container to NOT create IMEX channel device nodes automatically.

Reply:
@klueska I was thinking the `DisableImexChannelCreation` actually reads better than `DisableIMEXChannelCreation`. Do you have thoughts on this?

Reply:
Yes, I agree with `DisableImexChannelCreation`. It's the same rationale we had for writing `Mig` instead of `MIG`.